app/views/pages/get-started/contact-us-tutorial/building-the-ui.liquid

```diff
@@ -0,0 +1,99 @@
+---
+metadata:
+  title: Building the User Interface
+  description: Follow these step-by-step tutorials to build a Contact Us form on platformOS
+converter: markdown
+---
+
+To implement the _Contact Us_ form, we'll create a homepage with two fields: `email` and `body`, where users can enter their contact information. 
+
+{% include 'alert/tip', content: 'Creating a homepage is a bit special compared to creating a regular [page](/developer-guide/pages/pages). For detailed documentation, refer to the [Homepage](/developer-guide/pages/pages#homepage) documentation.' %}
+
+
+## Requirements
+
+* Followed all steps of the [Quick Setup Guide](/get-started/contact-us-tutorial/#quick-setup-guide) for building 
+* HTML knowledge
+* [Pages](/developer-guide/pages/pages)
```

Comment:
Reading 'pages'? Extensive routing knowledge? Lightly taking a look on this page? Do we really need it at all at this point? We'll just create a file in the next step.

Reply:
You are right! I'm removing it from the list, as the tutorial guide the user though the process without needing any prerequisites.